src/Disks/IO/AsynchronousReadIndirectBufferFromRemoteFS.cpp

```diff
@@ -354,6 +357,15 @@ off_t AsynchronousReadIndirectBufferFromRemoteFS::seek(off_t offset, int whence)
 }
 
 
+off_t AsynchronousReadIndirectBufferFromRemoteFS::getPosition()
+{
+    off_t res = static_cast<off_t>(file_offset_of_buffer_end - available() + bytes_to_ignore);
+    if (prefetch_future.valid())
+        res += static_cast<off_t>(last_prefetch_info.bytes_ignored);
```

Comment:
so if we had `seek -> prefetch -> getPosition`, then `bytes_to_ignore` == `bytes_ignored` and we will sum up extra `bytes_ignored` because the prefetch is valid? 
I agree about adding `bytes_to_ignore` is needed here, but seems like case with adding `last_prefetch_info.bytes_ignored` is redundant, I do not understand why it is needed.

Reply:
> then `bytes_to_ignore` == `bytes_ignored`

No, we reset `bytes_to_ignore` to 0 when starting prefetch, in `asyncReadInto()`.

(But do we have to reset it? Hmm)